datasets/nasa-power.yaml

```diff
@@ -0,0 +1,128 @@
+  POWER also provides parameters at daily, monthly, annual, and user-defined time periods, spanning from 1984 through to within a week of real time. Additionally, POWER provides are user-defined analytic capabilities, including custom climatologies and climatological-based reports for parameter anomalies, ASHRAE® compatible climate design condition statistics, and building climate zones. <br/><br/>
+  The ARD and climate analytics will be readily accessible through POWER's integrated services suite, including the [Data Access Viewer (DAV)](https://power.larc.nasa.gov/data-access-viewer/). The DAV has recently been improved to incorporate updated parameter groupings, new analytical capabilities, and the new data formats.  POWER also provides a complete [API (Application Programming Interface)](https://power.larc.nasa.gov/api/pages/) that allows uses to obtain all the parameters in the DAV plus additional parameters for larger repetitive orders or from within the user\'s own analytic tools. <br/><br/>
+Documentation: https://power.larc.nasa.gov/docs/
+Contact: larc-power-project@mail.nasa.gov
+ManagedBy: "[NASA POWER](https://power.larc.nasa.gov/)"
+UpdateFrequency: | 
+  Near Real Time (NRT); as soon as source data becomes available from our source data providers. <br/><br/>
+  - Meteorological: ~2-3 days <br/>
+  - Solar ~5-7 days <br/><br/>
+  For more information see the [NASA POWER Availability Dashboards](https://power.larc.nasa.gov/dashboard/availability/)
+Tags:
+  - agriculture
+  - air quality
+  - analytics
+  - archives
+  - atmosphere
+  - climate
+  - climate model
+  - data assimilation
+  - deep learning
+  - earth observation
+  - energy
+  - environmental
+  - forecast
+  - geoscience
+  - geospatial
+  - global
+  - netcdf
+  - history
+  - imaging
+  - industry
+  - machine learning
+  - machine translation
+  - metadata
+  - meteorological
+  - model
+  - opendap
+  - radiation
+  - satellite imagery
+  - solar
+  - statistics
+  - sustainability
+  - time series forecasting
+  - water
+  - weather
+  - zarr
+License: There are no restrictions on the use, access, and/or download of data from the NASA POWER Project. We request that you site the NASA POWER Project when using the data provided from NASA POWER Project.
+Resources:
+  - Description: POWER's Zarr Analysis Ready Data (ARD) Datasets
+    ARN: arn:aws:s3:::power-analysis-ready-datastore
+    Region: us-west-2
+    Type: S3 Bucket
+    RequesterPays: false
+    Explore:
+    - "[Browse Bucket](https://power-analysis-ready-datastore.s3.us-west-2.amazonaws.com/index.html)"
+  - Description: POWER's NetCDF Datastore
+    ARN: arn:aws:s3:::power-datastore
+    Region: us-west-2
+    Type: S3 Bucket
+    RequesterPays: false
+    Explore:
+    - "[Browse Bucket](https://power-datastore.s3.us-west-2.amazonaws.com/index.html)"
+DataAtWork:
```

Comment:
Just to mention -- this is a really nice collection of information including tutorials!